src/sonic-yang-models/yang-models/sonic-mux-linkmgr.yang

```diff
@@ -98,6 +98,20 @@ module sonic-mux-linkmgr {
                     description "Linkmgrd log verbosity level. ";
                 }
             }
+
+            container SERVICE_MGMT {
+
+                leaf kill_radv {
+                    type enumeration {
```

Comment:
Please use boolean instead.

Reply:
https://github.com/sonic-net/sonic-buildimage/blob/ea34cb2f0a92b32149369c2f61bf33c3fa25ada4/files/scripts/service_mgmt.sh#L43-L46

Hi Gang - we have capitalized letters so I can't make the field as boolean.